test-core-development.cfg

```diff
@@ -0,0 +1,10 @@
+[buildout]
+extends =
+    https://raw.githubusercontent.com/4teamwork/ftw-buildouts/master/test-package.cfg
+    https://raw.githubusercontent.com/4teamwork/ftw-buildouts/master/test-versions.cfg
```

Comment:
@lukasgraf I've added hotfixes to both test buildout. Waiting for tests in `opengever.ftw`.